config-generator/src/main/resources/infinispan.xml

```diff
@@ -38,11 +38,12 @@
                     <gsp:scriptlet>
                         <![CDATA[
                         if (keystore && keystore.path) {
+                            def selfSignAttr = keystore.selfSignCert ? "localhost" : ""
                         ]]>
                     </gsp:scriptlet>
                     <server-identities>
                         <ssl>
-                            <keystore path="${keystore.path}" keystore-password="${keystore.password}" alias="${keystore.alias}"/>
+                            <keystore path="${keystore.path}" alias="${keystore.alias}" keystore-password="${keystore.password}" generate-self-signed-certificate-host="${selfSignAttr}"/>
```

Comment:
Does an empty "generate-self-signed-certificate-host" behave the same as if it does not exist at all? I was thinking that we would have `selfSignAttr` equal to the attribute and value, that way if `!selfSignCert` the attribute just doesn't exist.

Reply:
The only sintax that seems to works with attribure is : `attribute="${expression}"` the forms like `${attributeplusvalue-expression}` breaks the syntax of the initial XML template.

Reply:
@rigazilla Is an empty "generate-self-signed-certificate-host" attr valid? If so, I don't mind that being present in the xml.

Reply:
yeah seems that it doesn't hurt Infinispan